packages/server-wallet/src/protocols/__test__/defunder.test.ts

```diff
@@ -0,0 +1,119 @@
+import {defaultTestConfig} from '../..';
+import {testKnex} from '../../../jest/knex-setup-teardown';
+import {MockChainService} from '../../chain-service';
+import {DBAdmin} from '../../db-admin/db-admin';
+import {createLogger} from '../../logger';
+import {AdjudicatorStatusModel} from '../../models/adjudicator-status';
+import {Channel} from '../../models/channel';
+import {Funding} from '../../models/funding';
+import {Store} from '../../wallet/store';
+import {TestChannel} from '../../wallet/__test__/fixtures/test-channel';
+import {Defunder} from '../defunder';
+
+const testChannelObj = TestChannel.create({
```

Comment:
Why is this `testChannelObj`?

Does `Obj` mean `Object`? I originally read it to mean `Objective`. I think both are confusing to me, but maybe `testChannelObject` is not confusing?

Reply:
I agree that is confusing. Changed in 7f30e0e86